src/modules/uORB/uORB.h

```diff
@@ -52,6 +52,7 @@ struct orb_metadata {
 	const uint16_t o_size;		/**< object size */
 	const uint16_t o_size_no_padding;	/**< object size w/o padding at the end (for logger) */
 	const char *o_fields;		/**< semicolon separated list of fields (with type) */
+	uint8_t o_id;			/**< ORB_ID enum */
```

Comment:
Why not
```suggestion
	ORB_ID o_id;			/**< ORB_ID enum */
```
?

Reply:
ORB_ID is an enum class and there's still a tiny amount of orb usage from C code. Removing it upstream is probably trivial, but we'd have to be willing to deprecate C support. There's also header dependencies to work out, but I'm sure that's solvable.

Reply:
To clarify, I'd actually like to consider dropping C support for uORB (and parameters for that matter), but I'm not quite ready to propose that.